src/pages/sta/cartesian_coordinates.astro

```diff
@@ -7,15 +7,297 @@ import BlueText from "../../components/BlueText.astro"
+          <th>range</th>
+          <th>definition</th>
+        </tr>
+        <tr>
+          <td>\(x\)</td>
+          <td>horizontal</td>
+          <td>\(-\infty \lt x \lt \infty\)</td>
+          <td>horizontal distance from the \(y\)-axis</td>
+        </tr>
+        <tr>
+          <td>\(y\)</td>
+          <td>vertical</td>
+          <td>\(-\infty \lt y \lt \infty\)</td>
+          <td>vertical distance from the \(x\)-axis</td>
+        </tr>
+      </table>
+    </Center>
+      <p>
+        In 3D the \(x\),\(y\),\(z\) axes are oriented in <em>right-handed
+        order</em>, so that fingers curling from \(x\) to \(y\) means
+        the thumb points in the \(z\) direction.
+      </p>
+    <Center>
+        <PrairieDrawCanvas id="rvp-fc-c" width="450" height="450">
+            <p class="figureCaption">
+                Cartesian coordinate system, showing the origin \(O\) and
+                three positions with their coordinates.
+            </p>
+        </PrairieDrawCanvas>
+    </Center>
+
+    <CalloutContainer slot="cards">
+        <CalloutCard title="Notation note">
+            <p>
+                When writing coordinates we can either list the coordinate
+                values explicitly, like \(x = 2\), \(y = 1\), or we can just
+                write the numbers in an ordered list, like \( (2,
+                1) \).
+              </p>
+              <p>
+                Writing them as a list assumes that we know which
+                coordinate system we are using, and which order the
+                coordinates are in. We will typically write the coordinate
+                values explicitly to avoid confusion. This also applies to
+                <a href="rvv.html#rvv-ic">vector components</a>.
+              </p>
+        </CalloutCard>
+
+        <CalloutCard>
+            <p>
+                While the idea of \(x,y\) coordinates specifying positions
+                might seem obvious, the use of algebraic quantities to
+                define geometry in this way was a major discovery of <a
+                href="http://en.wikipedia.org/wiki/Ren%C3%A9_Descartes">René
+                Descartes</a> and a key event in the <a
+                href="http://en.wikipedia.org/wiki/Scientific_Revolution">scientific
+                revolution</a>. Descartes was also instrumental in
+                establishing the philosophical groundwork of <a
+                href="http://en.wikipedia.org/wiki/Age_of_Enlightenment">The
+                Enlightenment</a>, and we owe to him the famous phrase <a
+                href="http://en.wikipedia.org/wiki/Cogito_ergo_sum">“<em>Cogito
+                ergo sum</em>”</a>.
+              </p>
+        </CalloutCard>
+    </CalloutContainer>
+</SubSection>
 
 </Section>
 
 <SubSection title="Vector Bases">
 
-<BlueText>Insert the same content here as from the TAM212 Reference Pages (Vectors and Bases - Vector Bases)</BlueText>
+<SubSection title="Vectors bases">
```

Comment:
I think we have an extra `SubSection` here? We're already inside a very similar `SubSection`.